packages/client/components/NewMeetingSidebarUpgradeBlock.tsx

```diff
@@ -0,0 +1,74 @@
+import styled from '@emotion/styled'
+import React from 'react'
+import {useHistory} from 'react-router'
+import {PALETTE} from '../styles/paletteV3'
+import FlatPrimaryButton from './FlatPrimaryButton'
+
+const RootBlock = styled('div')({
+  padding: '16px 8px'
+})
+
+const Container = styled('div')({
+  background: PALETTE.SLATE_200,
+  border: `2px solid ${PALETTE.GRAPE_500}`,
+  padding: 12,
+  borderRadius: 4
+})
+
+const Heading = styled('div')({
+  color: PALETTE.SLATE_700,
+  fontSize: '14px',
+  lineHeight: '16px',
+  fontWeight: 600
+})
+
+const Description = styled('div')({
+  color: PALETTE.SLATE_900,
+  fontSize: '12px',
+  lineHeight: '16px',
+  marginTop: 8
+})
+
+const Action = styled('div')({
+  marginTop: 8
+})
+
+const UpgradeButton = styled(FlatPrimaryButton)({
+  width: '100%'
+})
+
+interface Props {
+  onClick?: () => void
+  orgId: string
+}
+
+const NewMeetingSidebarUpgradeBlock = (props: Props) => {
+  const {onClick, orgId} = props
+  const history = useHistory()
+  return (
+    <RootBlock>
+      <Container>
+        <Heading>🎉 We’re glad you love Parabol!</Heading>
+
+        <Description>
+          To make sure you don’t lose access, make sure to upgrade to the Team plan so you can have
```

Comment:
+1 It might be helpful to explicitly state why they might lose access. While the end of the sentence explains that a benefit of upgrading is getting "as many teams as you need", it's not super clear to me why the user might lose access. Perhaps: "You've exceeded the two-team limit. To make sure you don't lose access, upgrade to the Team plan so you can have as many teams as you need."

I realise that this sentence was included in the designs, so feel free to ignore. Curious to get your thoughts though

Reply:
Make sense. I updated the copy. 
cc @acressall 

<img width="300" alt="image" src="https://user-images.githubusercontent.com/466991/205876435-4b9caa0c-af6c-4610-896d-4e817e7e9b58.png">